locale/en/blog/module/service-logging-in-json-with-bunyan.md

```diff
@@ -8,31 +8,28 @@ slug: service-logging-in-json-with-bunyan
 layout: blog-post.hbs
 ---
 
-<div style="float:right;margin:0 0 15px 15px;">
-<img class="alignnone size-full wp-image-469" title="Bunyan" src="https://nodeblog.files.wordpress.com/2012/03/bunyan.png" alt="Paul Bunyan and Babe the Blue Ox" width="240" height="320" /><br />
-<a href="http://www.flickr.com/photos/stublag/2876034487">Photo by Paul Carroll</a>
-</div>
+<div style="float:right;margin:0 0 15px 15px;">![Paul Bunyan and Babe the Blue Ox](https://nodeblog.files.wordpress.com/2012/03/bunyan.png "Bunyan")  
```

Comment:
Maybe add newlines before and after raw HTML. This can cause rendering issues in some cases.

Reply:
Checked with the local build, and even with the extra lines, it wasn't converting it back to HTML, so I just reverted this line